lisp/bazel-build.el

```diff
@@ -24,38 +24,52 @@
 
 ;;; Code:
 
+(defgroup bazel-build 'nil
```

Comment:
nit: nil is self-quoting, you don't have to quote it here

Reply:
Fixed.